plugins/globalias/globalias.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+globalias() {
+   zle _expand_alias
+   zle expand-word
+   zle self-insert
```

Comment:
I'd get rid of the self-insert here. That way you can do `....[space]` and continue making the path.